pkg/trace/config/config.go

```diff
@@ -485,12 +487,14 @@ func New() *AgentConfig {
 		RareSamplerCooldownPeriod: 5 * time.Minute,
 		RareSamplerCardinality:    200,
 
+		ReceiverEnabled:        true,
```

Comment:
Always true except when manually overridden. The idea behind this flag is to allow OTLP component to disable the entire receiver (for example [here](https://github.com/DataDog/datadog-agent/blob/main/comp/otelcol/otlp/components/statsprocessor/agent.go#L47-L48) )